installation/local/roles/web_install/tasks/apache2.yml

```diff
@@ -14,17 +14,40 @@
 - name: Apache - Self Signed Cert
   notify:
     - Restart Apache2
+  when: cloud_snitch_web_ssl_cert is not defined and cloud_snitch_web_ssl_key is not defined
```

Comment:
should this be an or statement? or maybe add a check to since
we can't really have one without the other

Reply:
I don't think we should default to a self signed cert on production if only one of cloud_snitch_web_ssl_cert or cloud_snitch_web_ssl_key is set.